tests/empty/jsfmt.spec.js

```diff
@@ -1 +1,12 @@
-run_spec(__dirname, ["flow", "typescript"]);
+const core = require("../../src/main/core-options");
```

Comment:
See #4554 about importing code directly in tests

Reply:
Might try using `require('../tests_config/require_prettier')` (or `require('prettier/local')`, who lands first) and `prettier.getSupportInfo()`